app/src/main/res/color/overlay_button_selector.xml

```diff
@@ -6,7 +6,6 @@
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:state_enabled="false" android:color="@color/button_disabled"/>
     <item android:state_pressed="true" android:color="@color/button_pressed" />
-    <item android:state_focused="true" android:color="@color/button_focused" />
```

Comment:
Okay, so you're deleting this line because in this case, focusing a button doesn't change it's background color ✅